app/src/pages/[resourceType]/search-result.tsx

```diff
@@ -24,9 +25,9 @@ export default function ResourceSearchResultsPage({
       </Center>
```

Comment:
The `"Search Returned" header` is also a bit off-centered from the two tabs (Repository/Authoring) in the navigation bar.